templates/zerver/help/import-users-and-channels-from-slack.md

```diff
@@ -0,0 +1,41 @@
+# Import Users and Channels from Slack
+
+{!follow-steps.md!} import users and channels from Slack to Zulip:
+
+1. Generate a Slack API token using Slack's [test token generator](https://api.slack.com/docs/oauth-test-tokens)
+   to import all of the necessary data.
+
+2. {!go-to-the.md!} **Your bots** {!settings.md!}
+
+3. Click on **Show/change your API key** button.
+
+4. Upon clicking the **Show/change your API key** button,
+   you will be asked to confirm your identity by entering
+   your password in the **Current password** field.
+
+5. Click the **Get API Key** button and copy generated API Key.
+
+6. Fill in all settings in `api/integrations/slack/zulip_slack_config.py`:
+
+    `SLACK_TOKEN` - token from point number 1.
+
+    `ZULIP_USER` - e-mail of the user (the user that API key was generated for).
+
+    `ZULIP_KEY` - API key from point number 4.
+
+    `ZULIP_SITE` - Zulip API server URI.
+
+    !!! warn "" Please ensure the user has admin rights.
+
+7. If imported users are not from the same organization you should change appropriate setting
+    {!go-to-the.md!} [Organization settings](/#administration/organization-settings) {!admin.md!}
+    Then uncheck marked setting
+    ![Administration](/static/images/help/administration-tab-organization-restriction.png)
```

Comment:
Instead of showing the option in this image can you please delete this image and step and move this to a separate section below step 9:

```.md
## Importing users from a different organization

If the users are not from the same organization, you should change your organization settings accordingly.

{!go-to-the.md!} [Organization settings](/#administration/organization-settings)
{!admin.md!}

2. Disable the **New users restricted to the following domains** option.